doc/changelogs/CHANGELOG_V17.md

```diff
@@ -45,6 +45,26 @@
   * [io.js](CHANGELOG_IOJS.md)
   * [Archive](CHANGELOG_ARCHIVE.md)
 
+<a id="17.9.1"></a>
+
+## 2022-05-31, Version 17.9.1 (Maintenance), @ruyadorno
+
+### Notable Changes
+
+* Upgrade npm to 8.11.0
+* Update to OpenSSL 3.0.3
+
+### Commits
+
+* \[17fef6ed54] - deps: upgrade npm to 8.11.0 (<npm-cli+bot@github.com>) <https://github.com/nodejs/node/pull/43210>
```

Comment:
Formatting is off on all of the entries in the list of commits. Did you you changelog-maker?
e.g. 
```suggestion
* \[[`17fef6ed54`](https://github.com/nodejs/node/commit/17fef6ed54)] - **deps**: upgrade npm to 8.11.0 (<npm-cli+bot@github.com>) [#43210](https://github.com/nodejs/node/pull/43210)
```

Also entirely up to you as to whether the npm-robot "author" should be consistent across all the npm update entries (you could pick the .mailmap across from master which should remap the names for you if you use changelog-maker).

Reply:
hmm. Looks like we should either update the release guide or change the default for changelog-maker 🤔 
```
$ npx changelog-maker --group --start-ref v17.9.0 --filter-release
npx: installed 244 in 12.166s
* [17fef6ed54] - deps: upgrade npm to 8.11.0 (npm-cli+bot@github.com) https://github.com/nodejs/node/pull/43210
* [d0b53c0557] - deps: upgrade npm to 8.10.0 (npm-cli+bot@github.com) https://github.com/nodejs/node/pull/43061
* [72630d176f] - deps: upgrade npm to 8.9.0 (npm-robot) https://github.com/nodejs/node/pull/42968
* [93d58c361e] - deps: upgrade npm to 8.8.0 (npm-robot) https://github.com/nodejs/node/pull/42886
* [34e6edd421] - deps: upgrade npm to 8.7.0 (ops+robot@npmjs.com) https://github.com/nodejs/node/pull/42744
* [02f8b0cdad] - deps: upgrade npm to 8.6.0 (npm team) https://github.com/nodejs/node/pull/42550
* [871eace946] - deps: update archs files for quictls/openssl-3.0.3 (RafaelGSS) https://github.com/nodejs/node/pull/43025
* [05fb807ab0] - deps: upgrade openssl sources to quictls/openssl-3.0.3 (RafaelGSS) https://github.com/nodejs/node/pull/43025
$ npx changelog-maker --group --start-ref v17.9.0 --filter-release --format markdown
npx: installed 244 in 6.33s
* \[[`17fef6ed54`](https://github.com/nodejs/node/commit/17fef6ed54)] - **deps**: upgrade npm to 8.11.0 (<npm-cli+bot@github.com>) [#43210](https://github.com/nodejs/node/pull/43210)
* \[[`d0b53c0557`](https://github.com/nodejs/node/commit/d0b53c0557)] - **deps**: upgrade npm to 8.10.0 (<npm-cli+bot@github.com>) [#43061](https://github.com/nodejs/node/pull/43061)
* \[[`72630d176f`](https://github.com/nodejs/node/commit/72630d176f)] - **deps**: upgrade npm to 8.9.0 (npm-robot) [#42968](https://github.com/nodejs/node/pull/42968)
* \[[`93d58c361e`](https://github.com/nodejs/node/commit/93d58c361e)] - **deps**: upgrade npm to 8.8.0 (npm-robot) [#42886](https://github.com/nodejs/node/pull/42886)
* \[[`34e6edd421`](https://github.com/nodejs/node/commit/34e6edd421)] - **deps**: upgrade npm to 8.7.0 (<ops+robot@npmjs.com>) [#42744](https://github.com/nodejs/node/pull/42744)
* \[[`02f8b0cdad`](https://github.com/nodejs/node/commit/02f8b0cdad)] - **deps**: upgrade npm to 8.6.0 (npm team) [#42550](https://github.com/nodejs/node/pull/42550)
* \[[`871eace946`](https://github.com/nodejs/node/commit/871eace946)] - **deps**: update archs files for quictls/openssl-3.0.3 (RafaelGSS) [#43025](https://github.com/nodejs/node/pull/43025)
* \[[`05fb807ab0`](https://github.com/nodejs/node/commit/05fb807ab0)] - **deps**: upgrade openssl sources to quictls/openssl-3.0.3 (RafaelGSS) [#43025](https://github.com/nodejs/node/pull/43025)
$
```